CardForm/src/main/java/com/braintreepayments/cardform/view/CardForm.java

```diff
@@ -181,6 +181,22 @@ public CardForm mobileNumberExplanation(String mobileNumberExplanation) {
         return this;
     }
 
+    /**
+     * @param masked if {@code true}, card number input will be masked.
+     */
+    public CardForm maskedCardNumber(boolean masked) {
```

Comment:
I think these methods should be named `maskCardNumber` and `maskCvv` instead of the past tense. Same goes for the param, `mask` instead of `masked`.